vignettes/GMAT-GMST.Rmd

```diff
@@ -0,0 +1,126 @@
+---
+title: "Example: Calculating GMST from GMAT in Hector"
+author: "Leeya Pressburger"
+date: "2021-08-18"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Example: Carbon tracking with Hector}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Introduction
+
+The global mean surface temperature (GMST) is the average of the air temperature over land and the temperature at the surface of the ocean.^[Allen, M.R., O.P. Dube, W. Solecki, F. Aragón-Durand, W. Cramer, S. Humphreys, M. Kainuma, J. Kala, N. Mahowald, Y. Mulugetta, R. Perez, M. Wairiu, and K. Zickfeld, 2018: Framing and Context. In: Global Warming of 1.5°C. An IPCC Special Report on the impacts of global warming of 1.5°C above pre-industrial levels and related global greenhouse gas emission pathways, in the context of strengthening the global response to the threat of climate change, sustainable development, and efforts to eradicate poverty [Masson-Delmotte, V., P. Zhai, H.-O. Pörtner, D. Roberts, J. Skea, P.R. Shukla, A. Pirani, W. Moufouma-Okia, C. Péan, R. Pidcock, S. Connors, J.B.R. Matthews, Y. Chen, X. Zhou, M.I. Gomis, E. Lonnoy, T. Maycock, M. Tignor, and T. Waterfield (eds.)]. In Press.] This is a very useful metric for observing global temperature changes. However, Hector does not produce GMST as an output at this time. Hector does measure the global mean air temperature (GMAT), which we can use to back-calculate GMST. GMAT is the air temperature measured 2 meters above the ground and is what is presented in a daily weather report.^[Taylor, Jessica. "Analyzing Surface Air Temperatures by Latitude." NASA, 2018. https://mynasadata.larc.nasa.gov/mini-lesson/analyzing-surface-air-temperatures-latitude-student-activity] 
```

Comment:
average of the air temperature over land and the temperature at the surface of the ocean. --> 
average of the air temperature 2m above the Earth's surface.

Reply:
Ooooh I didn't you could do footnotes like this. 👏